canonical-web-and-design/maas-ui
# feat(ui): add VLAN summary edit form

## Done

- Add a form to edit a VLAN on the VLAN details summary section.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the React VLAN details page.
- In the summary section there should be an Edit button. Click it.
- The edit form should appear.
- Change some details and click save. The form should close.

## Fixes

Fixes: canonical-web-and-design/app-tribe#667.
